### PR TITLE
lesson_1_1

### PR DESCRIPTION
**Pull request by daria1911**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,32 @@
...
-          Learn React
-        </a>
-      </header>
-    </div>
+      <>
+        <h1>Todo List</h1>
+        <ul>
+            {todoList.map( (item, idx) => {
+                return <li key={idx}>{item.title}</li>
+            })}
```
Comment (sinancengiz) [inline src/App.js]: Good job! only thing we need update is using item.id as key as instructed in the assignment. when we have an unique identifier for the item in the list we want to use it as a key, using index as key must be our last choose. 

also we can even shorten  map function as 
`{todoList.map( (item) => <li key={item.id}>{item.title}</li>)}`

please request another review from me when you made the changes

**Review by sinancengiz — CHANGES_REQUESTED** [review]
Great Job please see the comment and request another review when you done with the changes

File: src/App.js (hunk 1 above)
Comment (daria1911) [reply src/App.js]: fixed

**Review by sinancengiz — APPROVED** [review]
LGTM. Great Job!

**Pull request merged**